src/__tests__/directLine.test.ts

```diff
@@ -0,0 +1,19 @@
+import * as DirectLineExport from '../directLine';
+
+test('#setConnectionStatusFallback', () => {
+    const { DirectLine } = DirectLineExport;
+    expect(typeof DirectLine.prototype.setConnectionStatusFallback).toBe('function')
```

Comment:
How about moving this line below so we can do

```suggestion
    expect(typeof setConnectionStatusFallback).toBe('function');
```